### PR TITLE
Linh branch

### PR DESCRIPTION
**Pull request by linhnguyen3041999**
Hi Mr. Duy,
please review my code. It can't automatically merge

### CODE REVIEW

File: Convenience-store/src/main/java/com/convenience_store/repository/UserRepo.java [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
 
+import java.util.List;
+
 import org.springframework.data.jpa.repository.JpaRepository;
 
 import com.convenience_store.entity.User;
 
 public interface UserRepo extends JpaRepository<User, Long>{
-
+	List<User> findByIsDeleted(Boolean isDeleted);
```
Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/repository/UserRepo.java:10]: Need resolve conflict at this line before create PR. You should add your new code into this file normally.

File: Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.convenience_store.dto.UserDTO;
+import com.convenience_store.entity.User;
+import com.convenience_store.repository.UserRepo;
+import com.convenience_store.service.UserService;
+
+@Service
+public class UserServiceImpl implements UserService{
```
Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java:16]: Format code

File: Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	private ModelMapper mapper;  
+	
+	@Override
+	public List<UserDTO> findAll() {
+		List<User> users = repo.findByIsDeleted(true);
+				return users.stream()
+						.map(item -> mapper.map(item, UserDTO.class))
+						.collect(Collectors.toList());
+	}
+
```
Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java:31]: Remove blank line

**Review by nguyentranduy — CHANGES_REQUESTED** [review]

File: Convenience-store/src/main/java/com/convenience_store/repository/UserRepo.java [hunk 4]
```diff
@@ -1,9 +1,11 @@
 package com.convenience_store.repository;
 
+import java.util.List;
+
 import org.springframework.data.jpa.repository.JpaRepository;
 
 import com.convenience_store.entity.User;
 
 public interface UserRepo extends JpaRepository<User, Long>{
```
Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/repository/UserRepo.java:9]: Pls check format code

**Review by nguyentranduy — CHANGES_REQUESTED** [review]

File: Convenience-store/src/main/java/com/convenience_store/dto/UserDTO.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import com.convenience_store.entity.MemberShip;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDTO implements Serializable{
```
Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/dto/UserDTO.java:14]: Pls check format code

Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/dto/UserDTO.java:14]: Pls check format code

File: Convenience-store/src/main/java/com/convenience_store/repository/UserRepo.java (hunk 1 above)
Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/repository/UserRepo.java:10]: Pls check format code at line 9. And resolve conflict at line 10 (hold your new code).

File: Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java (hunk 2 above)
Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java:16]: Format code

File: Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java (hunk 3 above)
Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java:31]: Remove blank line

**Review by nguyentranduy — CHANGES_REQUESTED** [review]

File: Convenience-store/src/main/java/com/convenience_store/dto/UserDTO.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	
+	private static final long serialVersionUID = -6324251952619540157L;
+	
+	private Long id;
+	private String name;
+	private String email;
```
Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/dto/UserDTO.java:20]: Test cmt

Comment (nguyentranduy) [inline Convenience-store/src/main/java/com/convenience_store/dto/UserDTO.java:20]:
```suggestion
	private String email;
```
test cmt

File: Convenience-store/src/main/java/com/convenience_store/dto/UserDTO.java (hunk 5 above)
Comment (linhnguyen3041999) [reply Convenience-store/src/main/java/com/convenience_store/dto/UserDTO.java]: fixed format code

File: Convenience-store/src/main/java/com/convenience_store/repository/UserRepo.java (hunk 1 above)
Comment (linhnguyen3041999) [reply Convenience-store/src/main/java/com/convenience_store/repository/UserRepo.java]: fixed format code

File: Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java (hunk 2 above)
Comment (linhnguyen3041999) [reply Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java]: fixed format code

File: Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java (hunk 3 above)
Comment (linhnguyen3041999) [reply Convenience-store/src/main/java/com/convenience_store/service/impl/UserServiceImpl.java]: removed blank line

**Review by nguyentranduy — APPROVED** [review]
Ok, look good to merge

**Pull request merged**